data/borderlines.json

```diff
@@ -1,5 +1,41 @@
 {
   "cities": [
+    {
+      "name": "Biratnagar",
+      "latitude": 26.452,
+      "longitude": 87.271,
+      "timezone": "Asia/Kathmandu",
+      "officialLanguage": "Nepali",
+      "currency": {
+        "name": "Nepalese Rupee",
+        "code": "NPR"
+      },
+      "landArea": {
+        "value": 77,
+        "unit": "km²"
+      },
+      "elevation": {
+        "value": 73.728,
+        "unit": "meters"
+      },
+      "landmarks": [
+        {
+          "name": "Baraha Kshetra",
+          "latitude": 26.83,
+          "longitude": 87.18
+        },
+        {
+          "name": "Koshi Tappu Wildlife Reserve",
```

Comment:
Same for this, I don't think this falls within biratnagar

Reply:
I will check it and correct it.